devolutions-gateway/src/api/monitoring.rs

```diff
@@ -209,18 +212,76 @@ pub(crate) struct MonitorResult {
     request_start_time: OffsetDateTime,
     response_success: bool,
     #[serde(skip_serializing_if = "Option::is_none")]
-    response_messages: Option<String>,
+    response_message: Option<String>,
+    /// Duration it took for the response to be received, in seconds.
+    ///
+    /// On error, this value is set to f64::INFINITY.
     response_time: f64,
```

Comment:
@philippelatulippe What about using `Option` and returning `None` when there is no response time? The problem with infinity is that at the type level it looks like a regular value, and this tends to lead to subtle logic bugs.

Reply:
Yeah, I thought it was making sense too, but re-thinking about it…